content/en/news.md

```diff
@@ -3,6 +3,10 @@ title: News
 sidebar: false
 ---
 
+### SciPy 1.7.2 released
+
+_November 5, 2021_ -- SciPy 1.7.2 is a bug-fix release with no new features compared to 1.7.1.
```

Comment:
Can you add that this is the first release to add Python 3.10 support? That's an important feature I'd say!

Reply:
ok, I tried to revise the news item accordingly